test/e2e-common.sh

```diff
@@ -60,9 +60,16 @@ readonly BUCKETS=10
 LATEST_SERVING_RELEASE_VERSION=$(latest_version)
 
 # Latest net-istio release.
-LATEST_NET_ISTIO_RELEASE_VERSION=$(
-  curl -L --silent "https://api.github.com/repos/knative/net-istio/releases" | \
-    jq -r '[.[].tag_name] | sort_by( sub("knative-";"") | sub("v";"") | split(".") | map(tonumber) ) | reverse[0]')
+LATEST_NET_ISTIO_RELEASE_VERSION=$(align_net-istio_version "$LATEST_SERVING_RELEASE_VERSION")
+
+# Receives the latest serving version and searches for the same version with major and minor and searches for the latest patch
+function align_net-istio_version() {
+  local serving_version=$1
```

Comment:
The serving version can return a `knative-` prefix so we should drop it

```suggestion
  local serving_version=${1##knative-}
```

Reply:
Not necessary, since the tag_name in GitHub also contains the prefix "knative-" example:   ` "tag_name": "knative-v1.0.0"`therefore it works fine this way, by cutting away the last `.[#patch]`. I tested the function with ´knative-v1.0.0´ and ´knative-v1.0.1´ as an input and it worked fine.